packages/embark-utils/src/collections.ts

```diff
@@ -0,0 +1,10 @@
+export function last(array) {
+  return array[array.length - 1];
+}
+
+export function recursiveMerge(target, source) {
+  const merge = require('merge');
+  return merge.recursive(target, source);
```

Comment:
We could do it in one line 
```suggestion
  return require('merge').recursive(target, source);
```

Reply:
I actually didn't change this code and just moved it. However, if we do make changes here, I'd rather pull the `require()` to the top of the file instead of having them cluttered all over the place.

Will update.